src/app/tickets/ticket-list/ticket-list.component.html

```diff
@@ -1,20 +1,80 @@
-<h2 class="d-flex justify-content-center display-4">
-  Tickets Queued
-</h2>
-
-<div class="d-flex">
-  <button mat-raised-button color="primary" (click)="onNewTicket()">
-    Create Ticket
+<div fxLayout="row" fxLayoutAlign="center center" class="k-header">
+  <button
+    flFlex
+    mat-fab
+    color="primary"
+    matTooltip="Submit ticket"
+    (click)="openForm()"
+  >
+    <mat-icon>add</mat-icon>
   </button>
+  <div fxFlex class="k-header-title">Tickets</div>
 </div>
-<p>
-  TO DO - insert table with sort/filter of tickets instead of just listing
-  them...
-</p>
-<div class="p-2">
-  <app-ticket-item
-    *ngFor="let ticketEl of tickets; let i = index"
-    [ticket]="ticketEl"
-    [index]="i"
-  ></app-ticket-item>
+
+<div class="mat-elevation-z8" class="p-4">
+  <div fxLayout="row">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event)" placeholder="Ex. ium" />
+    </mat-form-field>
+    <mat-paginator
+      [pageSizeOptions]="[5, 10, 20]"
+      showFirstLastButtons
+      fxFlex
+    ></mat-paginator>
+  </div>
+
+  <table mat-table [dataSource]="tickets" class="w-100">
```

Comment:
Could set the value of the input to an [(ngModel)] and then set [dataSource]="methodThatAppliesTextFilter()" or better yet [dataSource]="tickets | customPipeApplyingTextFilter" (I think the better way?)

Reply:
I'll address this in the next ticket, I think the customPipe is the right way to go! Thanks